src/GraphQl/Resolver/Factory/CollectionResolverFactory.php

```diff
@@ -164,4 +167,22 @@ private function getSubresource(string $rootClass, array $rootResolvedFields, ar
             'collection' => $isCollection,
         ]);
     }
+
+    private function getNormalizedFilters($args)
+    {
+        $filters = $args;
+        foreach ($filters as $name => &$value) {
+            if ($name === $this->orderParameterName && \is_array($value)) {
```

Comment:
Why not generalizing it for all filters (i.e. only verifying if `$value` is an array)?

Reply:
My first idea was that this is not necessary fo all filters (like DateFilter) for example and wanted to keep the modification as small as possible... but on a second thought, I don't see any issue in generalizing it.

Reply:
And it simplifies the code :wink: